***_Please note, this table is being populated. If anyone interested for their paper to be included in this community resource, please send us your paper title_.**




**Review of Pharmaceutical Research**
| Authors |Data Type | Methodology|
| :---  | :--- | :--- |
[Zhavoronkov et al.](https://chemrxiv.org/ndownloader/articles/11829102/versions/2/export_pdf) [1]|Crystal structure,homology modelling,and co-crystallisedligands|The authors utilised generative models togenerate the molecules for the 3C-like protease that can actas potential inhibitors for SARS-CoV-2.|
[Hofmarcher et al.](https://arxiv.org/pdf/2004.00979) [2]|Drug-discovery databases|In this work, authors utilise ChemAI, a DNN trainedon million of data pints across 3.2 million of molecules, forscreening favourable inhibitors from the ZINC database for SARS-CoV-2.|
[Beck et al.](https://www.sciencedirect.com/science/article/pii/S2001037020300490) [3]|SMILES strings and amino acids sequences|Authors utilise a pre-trained drug-target interaction modelto predict commercially available antiviral drugs for COVID-19.|
[Kim et al.](https://chemrxiv.org/articles/Advanced_Bioinformatics_Rapidly_Identifies_Existing_Therapeutics_for_Patients_with_Coronavirus_Disease_-_2019_COVID-19_/12037416/files/22121172.pdf) [4]|SMILES strings and amino acids sequences|A binding affinity prediction platform is usedto detect available FDA approved drugs that can blockSARS-CoV-2 from entering cells.|
[Richardson et al.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7137985/) [5]|Biomedical data|Authors use BenevolentAI to search for approveddrugs that can block the viral infection process.|
[Stebbing et al.](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(20)30132-8/fulltext?rss=yes&utm_campaign=update-laninf&utm_source=hs_email&utm_medium=email&utm_content=84189611&_hsenc=p2ANqtz-8RghsAw_DEaPsvoGNQ2wQmlzBKQTBUp_wCgamJlQyjRnn7h99GsA7d1SndeGc-o6C1nWe9XWHrj5hGpuW9AE3aEV6bVg&_hsmi=84189611) [6]|Biomedical data|This study examines approved antiviral and anti-inflammatorytreatments for COVID-19.|
[Vijil et al.](https://arxiv.org/pdf/2004.01215) [7]|SMILES|Authors uses generative models to design drug candidates specific to agiven target protein sequence. They release around 3000 novel COVID-19 drug candidates.|




**References**

**[1]** Zhavoronkov, Alex, et al. "Potential COVID-2019 3C-like protease inhibitors designed using generative deep learning approaches." Insilico Medicine Hong Kong Ltd A 307 (2020).

**[2]** Hofmarcher, Markus, et al. "Large-scale ligand-based virtual screening for SARS-CoV-2 inhibitors using deep neural networks." Available at SSRN 3561442 (2020).

**[3]** Beck, Bo Ram, et al. "Predicting commercially available antiviral drugs that may act on the novel coronavirus (SARS-CoV-2) through a drug-target interaction deep learning model." Computational and Structural Biotechnology Journal (2020).

**[4]** Kim, Jason, et al. "Advanced Bioinformatics Rapidly Identifies Existing Therapeutics for Patients with Coronavirus Disease-2019 (COVID-19)." (2020).

**[5]** Richardson, Peter, et al. "Baricitinib as potential treatment for 2019-nCoV acute respiratory disease." Lancet (London, England) 395.10223 (2020): e30.

**[6]** Stebbing, Justin, et al. "COVID-19: combining antiviral and anti-inflammatory treatments." The Lancet Infectious Diseases 20.4 (2020): 400-402.

**[7]** Chenthamarakshan, Vijil, et al. "Target-Specific and Selective Drug Design for COVID-19 Using Deep Generative Models." arXiv preprint arXiv:2004.01215 (2020).

